src/Box.js

```diff
@@ -1,11 +1,14 @@
 import styled from 'styled-components'
-import { space, width, color } from 'styled-system'
+import { space, width, color, responsiveStyle } from 'styled-system'
 import PropTypes from 'prop-types'
 
+const visibility = responsiveStyle('visibility', 'visibility')
+
 const Box = styled.div`
   ${space}
   ${width}
   ${color}
+  ${visibility}
```

Comment:
I think rather than adding this to the Box component, which is intended to be used in a **lot** of situations, that this should be an **extension** of the Box component since responsively hiding elements is not as common – e.g.:

```jsx
const Hide = styled(Box)`
  ${responsiveHideUtility}
`
```